### ISSUE TITLE
Fiks Arkiv .NET client generates incorrect payload for Avskrivning in no.ks.fiks.arkiv.v1.arkivering.arkivmelding.opprett

### ISSUE THREAD

**Issue by myakimov** [ks-no/fiks-arkiv-specification#232]
As it seems using the Fiks Arkiv .NET client to generate the request arkivmelding for `no.ks.fiks.arkiv.v1.arkivering.arkivmelding.opprett` is generating the avskrivning child elements in the wrong schema associated with another message type - `no.ks.fiks.arkiv.v1.arkivering.avskrivning.opprett`.

Example of generated **korrespondencepart** element
```
<korrespondansepart>
      <korrespondansepartID>2689</korrespondansepartID>
      <korrespondanseparttype>
        <kode xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">IM</kode>
        <beskrivelse xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">Intern Mottaker</beskrivelse>
      </korrespondanseparttype>
      <erBehandlingsansvarlig>true</erBehandlingsansvarlig>
      <korrespondansepartNavn>Kristian</korrespondansepartNavn>
      <epostadresse>kristian@blabla.com</epostadresse>
      <administrativEnhet>
        <navn xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">Dokumentsenter</navn>
        <identifikator xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">8</identifikator>
        <initialer xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">DOKS</initialer>
      </administrativEnhet>
      <saksbehandler>
        <navn xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">Kristian</navn>
        <identifikator xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">16</identifikator>
        <initialer xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">KS</initialer>
      </saksbehandler>
      <erSkjermet>false</erSkjermet>
      <avskrivning>
        <avskrivningsdato xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/avskrivning/opprett/v1">2024-12-03</avskrivningsdato>
        <avskrevetAv xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/avskrivning/opprett/v1">Kristian</avskrevetAv>
        <avskrivningsmaate xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/avskrivning/opprett/v1">
          <kode xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">TE</kode>
          <beskrivelse xmlns="https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/metadatakatalog/v1">Tatt til etterretning</beskrivelse>
        </avskrivningsmaate>
      </avskrivning>
    </korrespondansepart>
```

The elements under `<avskrivning/>` should be under the namespace `https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/arkivmelding/opprett/v1` (arkivmelding), but a shown in the sample above they are in the `https://ks-no.github.io/standarder/fiks-protokoll/fiks-arkiv/avskrivning/opprett/v1` (avskrivning). As it seems in the .NET client there is a single Avskrivning class which is used in both places, but actually the `avskrivning` elment is in two distinc schemas

**Comment by jarleborsheim** [comment]
This was an error in the generation of code. It used the same namespace in C# for both avskrivning classes, and then the first generated class was overwritten by the last. Fixed in 1.0.4 version of the .NET nuget package.

**Comment by sturtzel** [comment]
Being tested now. Looks promising.
